botprise/README.md

```diff
@@ -0,0 +1,49 @@
+# Agent Check: botprise
+
+## Overview
+
+We have been using datadog Rest APIs to get usage metrics from devices. As part of integration we have installed datadog agent on few lab devices. With the help of agent we retrieve usage data from the devices and raise alert if usage cross threshold value. We have setup few monitors to check alerts for each host. For every alert we have been creating Service Now ticket which comes in botprise system with webhook integration.
+Later while resolving those tickets we use datadog rest API to get usage metrics and host details. Also APIs are being used for querying process details
```

Comment:
This should be an overview about what the integration is about not a backstory ;). It should include a link to the service official page